### DISCUSSION TITLE
Dịch random seed

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#95] [Terminology] [answered]
Từ này đang được mọi người giữ nguyên, nếu dịch thì dịch là `hạt giống` chăng?

cc @mlbvn/handson-ml

**Comment by ngcthuong** [comment]
Gọi là "nhân" thì sao nhỉ? "Nhân ngẫu nhiên"?

**Reply by minhduc0711** [reply]
Sao lại gọi là `nhân` anh nhỉ?

**Reply by hnguyentt** [reply]
Gọi `nhân` sợ bị lẫn lộn qua `kernel`.

**Reply by ngcthuong** [reply]
Từ nhân có lẽ là do bản thân mình :D. Trong đầu mình luôn dịch như vậy cho từ này.

**Comment by quangnhat185** [comment]
Mình vote giữ nguyên vì trong code cũng sử dụng các variable có tên là `random_seed` rồi

**Comment by cuongvng** [comment]
hm có vài chỗ đang để là 'seed ngẫu nhiên'.

**Reply by minhduc0711** [reply]
Mình thấy dịch từ `random` ra cũng được.

**Comment by ngcthuong** [comment]
Mình thấy mấy từ đơn giản như này không nên để nguyên. Thứ nhất vì nó đơn giản, thứ hai để lại quá nhiều từ tiếng Anh đơn giản nó gây ra cảm giác hổ lốn. Từ nào khó quá mình mới để nguyên thôi, chứ từ như này đâu đến nỗi vậy.

**Reply by minhduc0711** [reply]
Nếu dịch thì em prefer `hạt giống` cho gần với từ gốc.

**Comment by minhduc0711** [comment, marked as answer]
Mình đồng ý với anh Thuớng là từ nào đơn giản thì nên dịch, nên từ này cứ chốt là `hạt giống ngẫu nhiên` nhé.

**Comment by cuongvng** [comment]
"mầm ngẫu nhiên" thì sao nhỉ?

**Reply by minhduc0711** [reply]
mầm là cái này mà 🌱 :v
